modules/core/include/visp3/core/vpTime.h

```diff
@@ -85,4 +87,24 @@ VISP_EXPORT int wait(double t0, double t);
 VISP_EXPORT void wait(double t);
 };
 
+class VISP_EXPORT vpChrono
+{
+public:
+  vpChrono();
+
+  double getDurationMicros();
```

Comment:
Not sure about the naming:
 - `getDurationMicroseconds()`?
 - `getDurationMicro()`?
 - `getDurationUs()`?
 - `getDurationMs()`?
 - `getDurationMilli()`?
 - `getDurationMilliseconds()`?
 - ...

I just kept the same naming than for `vpTime::`

[Here](https://en.cppreference.com/w/cpp/chrono/duration) how time is defined in the standard library.

Reply:
Your naming sounds good to me